test/unit/spread.spec.ts

```diff
@@ -92,7 +92,7 @@ const spreadCases = [
     "1, 2, ...[3, 4, 5]",
     "...[1, 2], 3, 4, 5",
     "1, 2, ...'spread', 4, 5",
-    "1, [2], ...[3, 4, 5]",
+    "1, 2, ...[3, 4, 5], [2]",
```

Comment:
I have meant something like `[1, ...[[2]], 3]`, since both `[1].concat(2)` and `[1].concat([2])` are `[1, 2]`